drivers/gpu/drm/i915/display/intel_dp_aux.c

```diff
@@ -233,6 +233,10 @@ intel_dp_aux_xfer(struct intel_dp *intel_dp,
 		ch_data[i] = intel_dp->aux_ch_data_reg(intel_dp, i);
 
 	if (is_tc_port) {
+		if (!dig_port) {
+			ret = -ENXIO;
```

Comment:
out_fail just "return ret;", why not just return ret here and add additional out_fail?

Reply:
additionally suggest add some error log here